metro/proto/v1/spec.proto

```diff
@@ -123,6 +123,28 @@ service AdminService {
         };
     }
 
+    rpc FetchProjectCredentials(ProjectCredentials) returns (ProjectCredentials){
```

Comment:
Call it `GetProjectCredentials` ?

checked pubsub.proto, they use the keyword Get and List
https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto